app.js

```diff
@@ -66,51 +104,54 @@ csvFiles.forEach(file => {
         let createTable = `CREATE TABLE ${tableName} (`;
 
         // take care of any column contain " or ' or space 
-        if (colsName.join().indexOf(" ") > -1 || colsName.join().indexOf("'") > -1 || colsName.join().indexOf('"') > -1) {
-            for (let column of colsName) {
-
-                if (column.indexOf(" ") > -1 || column.indexOf("'") > -1 || column.indexOf('"') > -1) {
-                    createTable += '`' + column + '`' + " TEXT, ";
-                } else {
-                    createTable += column + " TEXT, ";
+        if (args[2] !== "--append") {
+            if (colsName.join().indexOf(" ") > -1 || colsName.join().indexOf("'") > -1 || colsName.join().indexOf('"') > -1) {
+                for (let column of colsName) {
+
+                    if (column.indexOf(" ") > -1 || column.indexOf("'") > -1 || column.indexOf('"') > -1) {
+                        createTable += '`' + column + '`' + " TEXT, ";
+                    } else {
+                        createTable += column + " TEXT, ";
+                    }
                 }
+                createTable = createTable.slice(0, -2) + ")";
+            } else {
+                createTable += colsName.join(" TEXT, ") + " TEXT)";
             }
-            createTable = createTable.slice(0, -2) + ")";
-        } else {
-            createTable += colsName.join(" TEXT, ") + " TEXT)";
         }
 
-        connection.query(createTable, error => {
-            if (error) {
-                console.error(error);
-                return;
-            }
+        if (args[2] === "--overwrite") {
```

Comment:
If I do `node app.js --files transactions.csv --overwrite` then it will not overwrite.
If I do `node app.js --files transactions.csv --append` then it will not append.
If I do `node app.js --append --files transactions.csv` then it will search the files folder for CSVs instead of using transactions.csv
And many more...
These are all incorrect behaviours.